modules/contact-form/grunion-contact-form.php

```diff
@@ -248,36 +248,47 @@ private static function register_contact_form_blocks() {
 
 		// Field render methods.
 		jetpack_register_block( 'field-text', array(
+			'parent'          => array( 'jetpack/contact-form' ),
 			'render_callback' => array( __CLASS__, 'gutenblock_render_field_text' ),
 		) );
 		jetpack_register_block( 'field-name', array(
+			'parent'          => array( 'jetpack/contact-form' ),
```

Comment:
as far as I can tell we would be passing this `parent` attribute to `register_block_type()` but in that context it doesn't mean anything. 

I think we should remove it before passing it into that function. See https://github.com/Automattic/jetpack/blob/add/contact-form-parent-fields/class.jetpack-gutenberg.php#L258.

Reply:
I'd actually prefer to keep it :slightly_smiling_face: I think it's fine to pass on this child/parent relation information to the block registry. Granted, it's not going to be used on the server side (for now), but I checked the [docs](https://developer.wordpress.org/reference/functions/register_block_type/) and it seemed fine to pass on arbitrary args. If it ever were to be used, I think it's fine since it faithfully reflects the value of the `parent` field in the client side `registerBlockType` calls.

Reply:
(And as discussed per DM, I'd rather not touch `Jetpack_Gutenberg` outside of #11091 since it'll mean more/trickier rebasing :sweat_smile: )